app/assets/stylesheets/results.scss

```diff
@@ -9,4 +9,7 @@ tr.solid-bottom-border {
 // make the confirmation panel a neutral colour
 .panel-blue {
   background: govuk-colour("blue");
-}
+  * {
+    color: white !important;
+  }
+}
```

Comment:
This looks quite extreme? Think a comment would be useful

Reply:
Good idea.